src/main/java/com/sforce/cd/apexUnit/client/testEngine/AsyncBulkApiHandler.java

```diff
@@ -114,9 +114,6 @@ public String handleBulkApiFlow(PartnerConnection conn, BulkConnection bulkConne
 			ApexUnitUtils
 					.shutDownWithDebugLog(e, "Caught IO exception while trying to deal with bulk connection: "
 							+ e.getMessage());
-		} catch (ConnectionException e) {
```

Comment:
The exception is handled within the getParentJobIdForTestQueueItems()  method itself now. Hence this change